restore.js

```diff
@@ -0,0 +1,74 @@
+import readline from 'readline';
+
+import * as firebase from 'firebase';
+import 'firebase/firestore';
+import fs from 'fs';
+
+function restoreDatabase() {
+    firebase.initializeApp(require('./firebase.test.config.json'));
+    const auth = firebase.auth();
+    const db = firebase.firestore();
+
+    const queries = {
+        'operations': db.collection('operations'),
```

Comment:
Can we just ask fb for all collections? Can we make this a generic package useful for any firebase user?